CredentialsManager/FilesExceptions/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace CredentialsManager.FilesExceptions
```

Comment:
It's a preference thing but this might be a bit too fine-grained of namespacing for my tastes.

Reply:
I'm new to the OOP and namespaces. At my level of experience  I like having everything organized in their own space.